packages/expect/src/__tests__/tsconfig.json

```diff
@@ -1,7 +1,5 @@
 {
   "extends": "../../../../tsconfig.test.json",
-  "compilerOptions": {
-    "rootDir": "../"
-  },
-  "include": ["../**/*"]
+  "include": ["./**/*"],
+  "references": [{"path": "../../"}]
```

Comment:
Feels like better setup. Please build `src` first and look at `__tests__` after. Also cleaner than using `rootDir`. Let’s see what CI thinks.